Source/Editor/CustomEditors/Dedicated/ActorEditor.cs

```diff
@@ -83,31 +112,93 @@ public override void Initialize(LayoutElementsContainer layout)
                     }
                 }
             }
+        }
 
-            base.Initialize(layout);
 
-            // Add custom settings button to General group
-            for (int i = 0; i < layout.Children.Count; i++)
-            {
-                if (layout.Children[i] is GroupElement group && group.Panel.HeaderText == "General")
-                {
-                    const float settingsButtonSize = 14;
-                    var settingsButton = new Image
-                    {
-                        TooltipText = "Settings",
-                        AutoFocus = true,
-                        AnchorPreset = AnchorPresets.TopRight,
-                        Parent = group.Panel,
-                        Bounds = new Rectangle(group.Panel.Width - settingsButtonSize, 0, settingsButtonSize, settingsButtonSize),
-                        IsScrollable = false,
-                        Color = FlaxEngine.GUI.Style.Current.ForegroundGrey,
-                        Margin = new Margin(1),
-                        Brush = new SpriteBrush(FlaxEngine.GUI.Style.Current.Settings),
-                    };
-                    settingsButton.Clicked += OnSettingsButtonClicked;
-                    break;
-                }
-            }
+        private void BetterGeneralEditor(LayoutElementsContainer group)
+        {
+            HorizontalPanelElement hor = group.HorizontalPanel();
+            hor.Panel.Margin = Margin.Zero;
+            hor.Control.Height = TextBoxBase.DefaultHeight * 2 + 2 + 2 + 2;
+
+            ImageElement img = hor.Image(Editor.Instance.Icons.Logo128);
```

Comment:
Okay, after thinking about this for a bit, I realized that for a proper actor-image/actor-icon implementation, we would need to be able to store editor-only metadata for every actor. I haven't looked too closely into this, but I think Flax currently doesn't have a way of doing that.

Reply:
Hmmm, well cant actor have like a SpriteHandle #IF FLAX_EDITOR or something like that?